test/integration/tasks.test.js

```diff
@@ -110,6 +110,7 @@ describe("Tasks", function () {
           expect(res.body.task.createdBy).to.equal(appOwner.username);
           expect(res.body.task.assignee).to.equal(appOwner.username);
           expect(res.body.task.participants).to.be.a("array");
+          expect(res.body.task.taskNumber).to.equal(3);
```

Comment:
❓Is this test a little brittle? How do we know this should be 3?

Reply:
In line number 34 https://github.com/Real-Dev-Squad/website-backend/pull/825/files#diff-8fd2efa3a5824bd99d050f5e6c5d5adffd59bf49902ac7bfb42bb8a7eaad4a52R34, we are creating a dummy task data array which has details of 2 tasks, so whenever we are making a post request shouldn't task number that is returned be 3 or should I check for it to be a number instead of just 3?